jans-cedarling/bindings/cedarling_c/tests/test_cedarling.c

```
@@ -667,28 +667,237 @@ void test_trusted_issuer_loading_info() {
+        "    },\n"
+        "    {\n"
+        "      \"resource\": {\"cedar_entity_mapping\": {\"entity_type\": \"Jans::Issue\", \"id\": \"c\"}, \"org_id\": \"some_long_id\", \"country\": \"US\"},\n"
+        "      \"action\": \"Jans::Action::\\\"UpdateForTestPrincipals\\\"\",\n"
+        "      \"context\": {}\n"
+        "    }\n"
+        "  ]\n"
+        "}";
+
+    CedarlingResult auth_result;
+    ret = cedarling_authorize_unsigned_batch(instance_id, mixed_batch, &auth_result);
+    TEST_ASSERT(ret == 0, "Mixed-item batch itself succeeds");
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)